### ISSUE TITLE
Giveth dashboard has no figures

### ISSUE THREAD

**Issue by clara-zi** [Giveth/giveth-dapps-v2#1644]
Giveth dashboard is blank and is it not refreshing.

**Comment by clara-zi** [comment]
![photo_2022-10-11_19-12-12](https://user-images.githubusercontent.com/100724132/195144663-6bcd4534-7df6-4828-a558-fc15c2309b1e.jpg)
![photo_2022-10-11_19-12-19](https://user-images.githubusercontent.com/100724132/195144668-df49a73c-1d34-4ccf-9297-61a10f937661.jpg)

**Comment by clara-zi** [comment]
It looks like the query is running ok behind the database but the data is not feeding through? @mateodaza

**Comment by clara-zi** [comment]
@mendesfabio - after our discussion about the dashboard recently - I would be really interested to learn the process of how the backend database links to the user interface - it could make sense for me to be responsible for maintaining the feed.

**Comment by clara-zi** [comment]
@mendesfabio: so this is a server issue (server is down) and Giveth info dashboard is not powered by Metabase. Thanks. @MoeNick

**Comment by laurenluz** [comment]
@divine-comedian - you're more caught up in the conversation about analytics & gov... do you know what we're doing here w/ the dashboard?

Is this possible/wroth fixing?

cc @MoeNick

**Comment by divine-comedian** [comment]
currently it's been almost impossible to coordinate between our dev teams and the giveth info team and the giveth info team doesn't seem very willing to work on this project anymore. 

TBH I would call giveth info a spectacular failure in terms of external coordination and this project is pretty much dead. IIRC @RamRamez made a donation dashboard that works pretty well - https://analytics-dashboard-blue.vercel.app/

**Comment by laurenluz** [comment]
yeah, that's what I thought. do you think we can just close this? @MoeNick is trying to just clean up the board.

**Comment by mendesfabio** [comment]
quite misleading to say we were not willing to work on this when we never got back from you on the last messages but yea I'm done feel free to close the issue.
